lisa/features/disks.py

```diff
@@ -2,12 +2,15 @@
 # Licensed under the MIT license.
 
 from functools import partial
+from pathlib import PurePosixPath
 from typing import Any, List, Optional, Type
 
 from assertpy.assertpy import assert_that
 
 from lisa import schema
 from lisa.feature import Feature
+from lisa.operating_system import CentOs
+from lisa.sut_orchestrator.azure.tools import Waagent
```

Comment:
It's not the right place. The `lisa/features` is neutral to platforms, so it shouldn't reference any platform. It can provide a method, and let platform to implement in subclasses.

Reply:
> It's not the right place. The `lisa/features` is neutral to platforms, so it shouldn't reference any platform. It can provide a method, and let platform to implement in subclasses.

Done